content/livesync/models/models.textile

```diff
@@ -93,30 +92,26 @@ SELECT COALESCE(MAX(sequence_id), 0) FROM outbox;
 COMMIT;
 ```
 
-For this to work correctly, use a "Read Committed":https://www.postgresql.org/docs/7.2/xact-read-committed.html transaction isolation level when reading your database state. This is the default isolation level in PostgreSQL.
+Use the *read committed transaction isolation level* for the correct function, when reading the database state. "PostgreSQL uses this":https://www.postgresql.org/docs/7.2/xact-read-committed.html as default. 
 
 h3(#history). Replay from history
 
-After the Models SDK initializes the model state via the sync function, it is important to apply the exact sequence of change events from your backend from the correct point in the stream. This position is specified by the @sequenceID@ returned from your backend endpoint.
+The SDK must initialize the model's state using the sync function and then precisely apply a sequence of change events from your backend, beginning at the correct point in the stream. This starting point is the @sequenceID@ your backend endpoint returns.
 
-Internally, after calling the sync function, the SDK attaches to the "channel":/channels and "subscribes":/channels to live messages. Live messages are buffered and not yet processed. It then paginates backwards through the message history on the channel from the point of attachment using "@untilAttach@":/storage-history/history#until-attach. The number of items in each page can be configured via the @syncOptions.historyPageSize@ defined in your "@ClientOptions@":/livesync/models#options.
+After the SDK executes the sync function, it internally connects to the "channel":/channels and begins "subscribing":/channels to live, yet unprocessed, messages.  It then paginates backward through the channel's message history, starting from the attachment point, using "@untilAttach@":/storage-history/history#until-attach. The number of items on each page can be configured via the @syncOptions.historyPageSize@ defined in your "@ClientOptions@":/livesync/models#options. 
 
-When a message with the target @sequenceID@ is reached, all of the messages from that point forward are sent to the model for processing to maintain continuity with incoming live messages.
+When a message is sent to the @sequenceID@, the model processes subsequent messages to maintain continuity with live messages. If the target @sequenceID@ is unreachable, there is insufficient message history to resume from the correct point to update the model. The SDK will attempt a re-sync. It re-syncs by calling the sync function again to acquire a newer version of the model state.
```

Comment:
> When a message is sent to the @sequenceID@

This one doesn't quite make sense, the sequence ID is not a recipient of a message :)